week-2/K-array-methods/exercise2.js

```diff
@@ -4,10 +4,10 @@
   The variable everyone should be an array containing both mentors and students.
 */
 
-var mentors = ["Daniel", "Irina", "Rares"];
+var mentors = ["Daniel", "Irina", "Rares", ","];
 var students = ["Rukmini", "Abdul", "Austine", "Swathi"];
 
-var everyone; // complete this statement
+var everyone = mentors + students; // complete this statement
```

Comment:
Unfortunately this isn't correct in JavaScript (it works in some other programming languages!).

To connect 2 arrays together we need to use the `.concat()` method.